snapshot-labs/sx-monorepo
# fix: show error on Editor composer textarea

### Summary



Laying the groundworks for #517 

The current proposal creation form does not validate the body, it just set a hard limit on the body textarea. 

In addition, the composer editor textarea is not styled exactly like the rest of the textarea, preventing using the same css to style the errors.

This PR will:

- add error support for the composer textarea, and validating the proposal body and user space statement
- refactor the editor textarea, to make use of the existing textarea styling

### Test 

- Go to the proposal creation page
- Fill the textarea over the 9600 chars limit
- It should show the error 

![Screenshot 2024-07-23 at 23 38 02](https://github.com/user-attachments/assets/cd2eef5a-bcd5-4054-b917-9b0f926a35e3)

The PREVIEW markdown tab default height has also been adjusted to match the textarea height by default

![Screenshot 2024-07-23 at 23 38 43](https://github.com/user-attachments/assets/cb1fcbb3-6b52-4a1d-b7fb-7fcf39754d61)
